config/daemon/ipv6.md

```diff
@@ -10,31 +10,110 @@ Before you can use IPv6 in Docker containers or swarm services, you need to
 enable IPv6 support in the Docker daemon. Afterward, you can choose to use
 either IPv4 or IPv6 (or both) with any container, service, or network.
 
-> **Note**: IPv6 networking is only supported on Docker daemons running on Linux
-> hosts.
+> **Note**
+>
+> - IPv6 support is experimental, use it with caution.
+> - IPv6 is only supported on Docker daemons running on Linux hosts.
 
-1.  Edit `/etc/docker/daemon.json`, set the `ipv6` key to `true` and the `fixed-cidr-v6` key
-    to your IPv6 subnet. In this example we are setting it to `2001:db8:1::/64`.
+## Daemon configuration
 
-    ```json
-    {
-      "ipv6": true,
-      "fixed-cidr-v6": "2001:db8:1::/64"
-    }
-    ```
+To enable IPv6, you must edit the Docker daemon configuration file located at
+`/etc/docker/daemon.json`. Configure the following parameters:
 
-    Save the file.
+```json
+{
+  "experimental": true,
+  "ipv6": true,
+  "ip6tables": true,
+  "fixed-cidr-v6": "2001:db8:1::/64",
+  ...
+}
+```
 
-2.  Reload the Docker configuration file.
+This configuration makes IPv6 networking function as you would expect it to.
+The `ipv6` and `fixed-cidr-v6` parameters are optional.
+They assign an IPv6 subnet to the default bridge network.
```

Comment:
My grammar isn't great, so I could be mistaken if a comma properly connects the two statements :sweat_smile: 

```suggestion
The `ipv6` and `fixed-cidr-v6` parameters are optional,
they assign an IPv6 subnet to the default bridge network.
```